xds/build.gradle

```diff
@@ -77,7 +74,6 @@ shadowJar {
     classifier = null
     dependencies {
         include(project(':grpc-xds'))
-        include(dependency('io.envoyproxy.protoc-gen-validate:'))
     }
     relocate 'com.github.udpa', 'io.grpc.xds.shaded.com.github.udpa'
     relocate 'io.envoyproxy', 'io.grpc.xds.shaded.io.envoyproxy'
```

Comment:
Seems like we would need a new relocation.

Reply:
Are you referring to shadowing PGV?
https://github.com/grpc/grpc-java/blob/ad5701815c6659968a5b47fb3c3cbe17bf5d3ed8/xds/third_party/protoc-gen-validate/src/main/proto/validate/validate.proto#L5

The parent package `io.envoyproxy` is relocated.

Reply:
Yeah. Okay. Sounds good.